empty

```diff
@@ -163,7 +163,8 @@ <h3><a href="#definitions"> 1.1 Definitions </a></h3>
 <p>
 <dt> <i>empty class</i> </dt>
 <dd>
-A class with no non-static data members other than zero-width bitfields,
+A class with no (named) non-static data members,
+no anonymous bit-field members other than zero-width bitfields,
```

Comment:
Sorry for the very delayed review.  I agree that we should fix the wording here, but I would slightly tweak the proposal:

* In the new clause, "anonymous bit-field members" should just be "unnamed bit-fields" (since "unnamed" is the term from the standard and since unnamed bit-fields are, as noted, not members).

* The "(named)" parenthetical invites confusion because there are other unnamed declarations that do create members (specifically, anonymous structs and unions).  If we feel the need to clarify this, we should add a longer and more explicit note to the new clause.

The standard says that an anonymous bit-field is not a member.  If we ignore that, I think the proposed change is redundant 

I'm not sure I understand how this proposed change actually changes anything.  I understand how the change in the standard's wording was necessary, but this change seems equivalent.  A bit-field is still a non-static data member

 it used to be `\not \exists m \in Members(T) . Variable(m) && !Static(m) &&